net-p2p/createtorrent/createtorrent-1.1.4-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils
+
+DESCRIPTION="Create BitTorrent files easily"
+HOMEPAGE="http://www.createtorrent.com/"
+SRC_URI="http://www.createtorrent.com/${PN}-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE=""
+
+DEPEND="dev-libs/openssl"
+RDEPEND="${DEPEND}"
+BDEPEND=""
+
+src_unpack() {
+	unpack ${A}
+	cd "${S}"
+	sed -i "s:[[]ssl[]]:[crypto]:" configure.in || die "sed failed..."
+	eautoreconf
+}
```

Comment:
https://devmanual.gentoo.org/ebuild-writing/functions/src_unpack/index.html
" Note: When using EAPI >=2, you should use the src_prepare function to apply patches or alter any of the source files, instead of the src_unpack. "